DecipherNow/gm-fabric-dashboard
# Refactor: Inspector SCSS

Resolves #1047 

## Proposed Changes

  - Refactored inspector.scss into styled-components
  - Updated folder structure
  - Deleted inspector.scss
  
## How to Test

  - Navigate to Explorer view and make sure the left column (search input/item list) is functioning as expected
